dbt/adapters/redshift/impl.py

```diff
@@ -27,13 +27,13 @@ class RedshiftConfig(AdapterConfig):
     sort: Optional[str] = None
     bind: Optional[bool] = None
     backup: Optional[bool] = True
+    autorefresh: Optional[bool] = False
 
 
 class RedshiftAdapter(SQLAdapter):
     Relation = RedshiftRelation
     ConnectionManager = RedshiftConnectionManager
     connections: RedshiftConnectionManager
-    Column = RedshiftColumn  # type: ignore
```

Comment:
We did this because we initially inherited from PostgresAdapter, however we wanted the default Column from `dbt-core`. So we redefined it in `dbt-redshift` and overwrote `PostgresAdapter.Column` with (effectively) `SQLAdapter.Column`. Now that we're back to inheriting from `SQLAdapter`, we no longer need this, nor the associated file/class.